sdk/eventhub/Azure.ResourceManager.EventHubs/tests/Tests/ScenarioTests.EventHubTests.CRUD.cs

```diff
@@ -52,14 +53,14 @@ public async Task EventCreateGetUpdateDelete()
             {
                 AccessTier = AccessTier.Hot
             };
-            await WaitForCompletionAsync(await StorageManagementClient.StorageAccounts.StartCreateAsync(resourceGroup, accountName, storageAccountCreateParameters));
+            await WaitForCompletionAsync(await StorageManagementClient.StorageAccounts.StartCreateAsync(resourceGroupName, accountName, storageAccountCreateParameters));
```

Comment:
This test case actually doesn't work (I tried on `main` branch). The following error will be thrown.

> "System.InvalidOperationException : Expected scope StorageAccountsOperations.StartCreate to be marked as failed > but it succeeded
> Exception: System.NotSupportedException: Parent does not have a default constructor. The default constructor must be explicitly defined."

Reply:
Since `Storage` is involved here, if we can confirm that we can put this test case aside, it will accelerate the publish of `Storage` mgmt track2 SDK preview as well.